DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -35,10 +35,25 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
 
             CourseContent? courseContent = null;
             return connection.Query<CourseContent, CourseSection, CourseContent>(
-                @"SELECT Customisations.CustomisationID AS id,
+                @"  WITH CustomisationDurations AS (

```

Comment:
Ah I didn't realise this would need changes as big as this! Just for the future it might have been a good idea to write a HLD for this or ping me a message to let me know this would require backend changes. Because in that case I might have suggested we push back on it or do a slightly different solution (e.g. replace `h` and `m` with hours and minutes with text replacement, possibly just in the voice over). We might very well have ended up with this solution anyway but would have been useful to consider 🙂

Reply:
Yeah neither did I! I spotted that the scalar function would need replacing early on but I had hoped I would get away with a simple `SUM` and then I kept realising how much more stuff I needed to add. Sorry I should have checked in earlier, but I didn't spend very much time on it if you did want to solve this problem a totally different way 🙂

Reply:
Okay, good to know 🙂 It would be a bit inconvenient to push back on right now as people are on holiday. So I think I'm happy to go with this solution, as long as we add some tests comparing it to the old function to make sure we haven't broken anything 🙂